contribute.html

```diff
@@ -56,11 +56,21 @@
       <article>
         <section>
           <h3>We'd love your help</h3>
-          <p>Zulip welcomes all forms of contributions! There's a ton of stuff to do. Here's just a small sampling of ways you can get started:</p>
+          <p>Zulip is licensed under the Apache license and is a
+          friendly open source community that welcomes all forms of
+          contributions! There's a ton of stuff to do -- in
+          development, testing, translation, documentation, and
+          more. Here's just a small sampling of ways you can get
+          started:</p>
           <ul>
-            <li>Write an integration for your favorite service</li>
-            <li>Write a docker file to make Zulip available to dockerized servers</li>
+            <li><a href="https://zulip.readthedocs.io/en/latest/integration-guide.html">Write
+            an integration for your favorite service</a></li>
+            <li>Improve
+            our <a href="https://github.com/zulip/zulip/blob/master/Dockerfile">Dockerfile</a>
+            to make Zulip available to dockerized servers</li>
             <li>Add a service monitoring dashboard</li>
+	    <li><a href="https://zulip.readthedocs.io/en/latest/translating.html">Translate</a>
+	    user-visible words into another language</li>
             <li>Add your favorite emoji</li>
```

Comment:
Let's get rid of emoji and dashboard and replace them with things like "contribute to the roadmap"

Or, thinking about this more, do you think we should just dump this page and make it a link to (potentially an improved version of) http://zulip.readthedocs.io/en/latest/readme-symlink.html#ways-to-contribute ?